src/org/opensolaris/opengrok/analysis/perl/PerlXref.lex

```diff
@@ -18,7 +18,7 @@
  */
 
 /*
- * Copyright (c) 2010, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2005, 2015, Oracle and/or its affiliates. All rights reserved.
```

Comment:
2010, 2015
in 2005 there was no perl analyzer, it was first public release with just C and Java